src/region_block.cpp

```diff
@@ -154,18 +155,21 @@ RegBlock::RegBlock(LAMMPS *lmp, int narg, char **arg) :
   if (ylo > yhi) error->all(FLERR, "Illegal region block ylo: {} >= yhi: {}", ylo, yhi);
   if (zlo > zhi) error->all(FLERR, "Illegal region block zlo: {} >= zhi: {}", zlo, zhi);
 
-  // extent of block
-
-  if (interior && !dynamic && !varshape) {
-    bboxflag = 1;
-    extent_xlo = xlo;
-    extent_xhi = xhi;
-    extent_ylo = ylo;
-    extent_yhi = yhi;
-    extent_zlo = zlo;
-    extent_zhi = zhi;
-  } else
-    bboxflag = 0;
+  // set extent of block
+
+  if (interior) {
```

Comment:
Is it true that `bboxflag` is always 1 if the surface is interior? Why set it equal to 1 separately in each branch of the if statement?

Reply:
> Is it true that bboxflag is always 1 if the surface is interior?

For the block region after the changes in this pull request, yes.

> Why set it equal to 1 separately in each branch of the if statement?

Copy-n-modify and then diff-n-merge, or in other words: laziness.